tests/internals/replica_update_so/replica_update_so_test.go

```diff
@@ -0,0 +1,322 @@
+//go:build e2e
+// +build e2e
+
+package replica_update_so
```

Comment:
```suggestion
package replica_update_so_test
```

Reply:
Seems like static checks complains about this naming (it is actually against go convention), it has been probably recently added to the linter: 
```
tests/internals/replica_update_so/replica_update_so_test.go:4:9: var-naming: don't use an underscore in package name (revive)
package replica_update_so
        ^
tests/internals/trigger_update_so/trigger_update_so_test.go:4:9: var-naming: don't use an underscore in package name (revive)
package trigger_update_so
```

@JorTurFer we should think about a solution how to change our naming convetion in e2e tests.

Reply:
also your suggestion doesn't match the dir name

Reply:
just adding _test is enough to silent the warning xD

Reply:
This role has been in the linter for more than a year 😄 We usually don't use underscores, the only place where we use them is for `_test`. All the other package names are without _

Reply:
@JorTurFer 🤦‍♂️ 😄 you are right! :) @gauron99 let's use this one

Reply:
added suffix to package names as `_test`